### ISSUE TITLE
Add additional HTTP headers

### ISSUE THREAD

**Issue by 0xfeedface** [AKSW/node_ldp#3]
- Cache-Control
- Last-Modified

**Comment by seebi** [comment]
- Allow
  (see http://www.w3.org/Submission/ldbp/#bpr-HTTP_HEAD)

**Comment by white-gecko** [comment]
Souldn't it also return a document type header?

It returnes:

```
$ netcat localhost 8001
GET http://webid.example.com/me
Accept: text/turtle

HTTP/1.1 200 OK
Date: Mon, 15 Apr 2013 16:15:07 GMT
Server: Crystal/0.1
Content-Length: 765
Connection: close

… (content)
```

**Comment by 0xfeedface** [comment]
Do you mean `Content-Type`?

**Comment by white-gecko** [comment]
Yes. But maybe netcat just doesn't show me the complete header (which would be strange).

**Comment by 0xfeedface** [comment]
There is currently no `Content-Type` header sent. But I don't know of any `Document-Type` header. That's why I'm asking.

**Comment by white-gecko** [comment]
sorry I meant `Content-Type`

**Comment by 0xfeedface** [comment]
For `ETag` we need a canonical representation, based on which we can compute the hash. RDF/JSON does not work because it does not enforce a specific ordering of properties and object values. Any ideas?

**Comment by seebi** [comment]
so we need to normalize it in some way.
rdf crypto tools use sorted ntriples for that (with bnode renaming / sorting as well)
